### ISSUE TITLE
"npm install" error on macOS + Node 14 environment

### ISSUE THREAD

**Issue by sanak** [Viglino/ol-ext#617]
`npm install` reports the following error on macOS Catalina + Node 14 (via nvm) environment.
```
% npm install
npm WARN rm not removing /Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/.bin/rimraf as it wasn't installed by /Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/rimraf
npm WARN rm not removing /Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/.bin/semver as it wasn't installed by /Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/semver

> fsevents@1.2.4 install /Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents
> node install

node-pre-gyp WARN Tried to download(404): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.4/fse-v1.2.4-node-v83-darwin-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for fsevents@1.2.4 and node@14.17.0 (node-v83 ABI, unknown) (falling back to source compile with node-gyp) 
  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
In file included from ../fsevents.cc:6:
In file included from ../../nan/nan.h:202:
In file included from ../../nan/nan_converters.h:67:
../../nan/nan_converters_43_inl.h:22:1: error: no viable conversion from 'Local<v8::Context>' to 'v8::Isolate *'
X(Boolean)
^~~~~~~~~~
../../nan/nan_converters_43_inl.h:18:23: note: expanded from macro 'X'
      val->To ## TYPE(isolate->GetCurrentContext())                            \
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:2848:37: note: passing argument to parameter
      'isolate' here
  Local<Boolean> ToBoolean(Isolate* isolate) const;
                                    ^
In file included from ../fsevents.cc:6:
In file included from ../../nan/nan.h:202:
In file included from ../../nan/nan_converters.h:67:
../../nan/nan_converters_43_inl.h:40:1: error: no viable conversion from 'Local<v8::Context>' to 'v8::Isolate *'
X(bool, Boolean)
^~~~~~~~~~~~~~~~
../../nan/nan_converters_43_inl.h:37:29: note: expanded from macro 'X'
  return val->NAME ## Value(isolate->GetCurrentContext());                     \
                            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:2858:30: note: passing argument to parameter
      'isolate' here
  bool BooleanValue(Isolate* isolate) const;
                             ^
In file included from ../fsevents.cc:6:
In file included from ../../nan/nan.h:203:
In file included from ../../nan/nan_new.h:189:
../../nan/nan_implementation_12_inl.h:103:42: error: no viable conversion from 'v8::Isolate *' to
      'Local<v8::Context>'
  return scope.Escape(v8::Function::New( isolate
                                         ^~~~~~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:190:7: note: candidate constructor
      (the implicit copy constructor) not viable: no known conversion from 'v8::Isolate *' to
      'const v8::Local<v8::Context> &' for 1st argument
class Local {
      ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:190:7: note: candidate constructor
      (the implicit move constructor) not viable: no known conversion from 'v8::Isolate *' to
      'v8::Local<v8::Context> &&' for 1st argument
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:194:13: note: candidate template ignored: could not
      match 'Local<type-parameter-0-0>' against 'v8::Isolate *'
  V8_INLINE Local(Local<S> that)
            ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:4446:22: note: passing argument to parameter
      'context' here
      Local<Context> context, FunctionCallback callback,
                     ^
In file included from ../fsevents.cc:6:
In file included from ../../nan/nan.h:203:
In file included from ../../nan/nan_new.h:189:
../../nan/nan_implementation_12_inl.h:337:37: error: too few arguments to function call, expected 2, have 1
  return v8::StringObject::New(value).As<v8::StringObject>();
         ~~~~~~~~~~~~~~~~~~~~~      ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:5854:3: note: 'New' declared here
  static Local<Value> New(Isolate* isolate, Local<String> value);
  ^
In file included from ../fsevents.cc:6:
../../nan/nan.h:1034:53: error: too few arguments to function call, single argument 'context' was not specified
      v8::Local<v8::String> string = from->ToString();
                                     ~~~~~~~~~~~~~~ ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:2810:3: note: 'ToString' declared here
  V8_WARN_UNUSED_RESULT MaybeLocal<String> ToString(
  ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8config.h:431:31: note: expanded from macro
      'V8_WARN_UNUSED_RESULT'
#define V8_WARN_UNUSED_RESULT __attribute__((warn_unused_result))
                              ^
In file included from ../fsevents.cc:6:
../../nan/nan.h:1044:37: error: cannot initialize a parameter of type 'v8::Isolate *' with an lvalue of type
      'char *'
        length_ = string->WriteUtf8(str_, static_cast<int>(len), 0, flags);
                                    ^~~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3037:26: note: passing argument to parameter
      'isolate' here
  int WriteUtf8(Isolate* isolate, char* buffer, int length = -1,
                         ^
In file included from ../fsevents.cc:6:
../../nan/nan.h:1818:28: error: no matching member function for call to 'Set'
    New(persistentHandle)->Set(New(key).ToLocalChecked(), value);
    ~~~~~~~~~~~~~~~~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3670:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
                                    ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3673:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
                                    ^
In file included from ../fsevents.cc:6:
../../nan/nan.h:1824:28: error: no matching member function for call to 'Set'
    New(persistentHandle)->Set(key, value);
    ~~~~~~~~~~~~~~~~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3670:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
                                    ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3673:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
                                    ^
In file included from ../fsevents.cc:6:
../../nan/nan.h:1830:28: error: no matching member function for call to 'Set'
    New(persistentHandle)->Set(index, value);
    ~~~~~~~~~~~~~~~~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3670:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
                                    ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3673:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
                                    ^
In file included from ../fsevents.cc:6:
../../nan/nan.h:1836:32: error: no matching member function for call to 'Get'
        New(persistentHandle)->Get(New(key).ToLocalChecked()));
        ~~~~~~~~~~~~~~~~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3717:43: note: candidate function not viable:
      requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3720:43: note: candidate function not viable:
      requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^
In file included from ../fsevents.cc:6:
../../nan/nan.h:1842:48: error: no matching member function for call to 'Get'
    return scope.Escape(New(persistentHandle)->Get(key));
                        ~~~~~~~~~~~~~~~~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3717:43: note: candidate function not viable:
      requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3720:43: note: candidate function not viable:
      requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^
In file included from ../fsevents.cc:6:
../../nan/nan.h:1847:48: error: no matching member function for call to 'Get'
    return scope.Escape(New(persistentHandle)->Get(index));
                        ~~~~~~~~~~~~~~~~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3717:43: note: candidate function not viable:
      requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3720:43: note: candidate function not viable:
      requires 2 arguments, but 1 was provided
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                          ^
In file included from ../fsevents.cc:6:
In file included from ../../nan/nan.h:2657:
../../nan/nan_object_wrap.h:24:25: error: no member named 'IsNearDeath' in 'Nan::Persistent<v8::Object,
      v8::NonCopyablePersistentTraits<v8::Object> >'
    assert(persistent().IsNearDeath());
           ~~~~~~~~~~~~ ^
/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/assert.h:93:25: note: expanded from macro
      'assert'
    (__builtin_expect(!(e), 0) ? __assert_rtn(__func__, __FILE__, __LINE__, #e) : (void)0)
                        ^
In file included from ../fsevents.cc:6:
In file included from ../../nan/nan.h:2657:
../../nan/nan_object_wrap.h:67:18: error: no member named 'MarkIndependent' in 'Nan::Persistent<v8::Object,
      v8::NonCopyablePersistentTraits<v8::Object> >'
    persistent().MarkIndependent();
    ~~~~~~~~~~~~ ^
../../nan/nan_object_wrap.h:124:26: error: no member named 'IsNearDeath' in 'Nan::Persistent<v8::Object,
      v8::NonCopyablePersistentTraits<v8::Object> >'
    assert(wrap->handle_.IsNearDeath());
           ~~~~~~~~~~~~~ ^
/Library/Developer/CommandLineTools/SDKs/MacOSX10.15.sdk/usr/include/assert.h:93:25: note: expanded from macro
      'assert'
    (__builtin_expect(!(e), 0) ? __assert_rtn(__func__, __FILE__, __LINE__, #e) : (void)0)
                        ^
In file included from ../fsevents.cc:6:
In file included from ../../nan/nan.h:2753:
../../nan/nan_typedarray_contents.h:34:43: warning: 'GetContents' is deprecated: Use GetBackingStore. See
      http://crbug.com/v8/9908. [-Wdeprecated-declarations]
      data   = static_cast<char*>(buffer->GetContents().Data()) + byte_offset;
                                          ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:5272:3: note: 'GetContents' has been explicitly
      marked deprecated here
  V8_DEPRECATE_SOON("Use GetBackingStore. See http://crbug.com/v8/9908.")
  ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8config.h:402:39: note: expanded from macro
      'V8_DEPRECATE_SOON'
# define V8_DEPRECATE_SOON(message) [[deprecated(message)]]
                                      ^
../fsevents.cc:50:32: error: no template named 'Handle' in namespace 'v8'
    static void Initialize(v8::Handle<v8::Object> exports);
                           ~~~~^
../fsevents.cc:63:6: warning: field 'async_resource' will be initialized after field 'lockStarted' [-Wreorder-ctor]
   : async_resource("fsevents:FSEvents"), lockStarted(false) {
     ^
In file included from ../fsevents.cc:82:
../src/constants.cc:89:11: error: no matching member function for call to 'Set'
  object->Set(Nan::New<v8::String>("kFSEventStreamEventFlagNone").ToLocalChecked(), Nan::New<v8::Integer>(kF...
  ~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3670:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
                                    ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3673:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
                                    ^
In file included from ../fsevents.cc:82:
../src/constants.cc:90:11: error: no matching member function for call to 'Set'
  object->Set(Nan::New<v8::String>("kFSEventStreamEventFlagMustScanSubDirs").ToLocalChecked(), Nan::New<v8::...
  ~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3670:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
                                    ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3673:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
                                    ^
In file included from ../fsevents.cc:82:
../src/constants.cc:91:11: error: no matching member function for call to 'Set'
  object->Set(Nan::New<v8::String>("kFSEventStreamEventFlagUserDropped").ToLocalChecked(), Nan::New<v8::Inte...
  ~~~~~~~~^~~
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3670:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
                                    ^
/Users/sanak/Library/Caches/node-gyp/14.17.0/include/node/v8.h:3673:37: note: candidate function not viable:
      requires 3 arguments, but 2 were provided
  V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
                                    ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
2 warnings and 20 errors generated.
make: *** [Release/obj.target/fse/fsevents.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/sanak/.nvm/versions/node/v14.17.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:376:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
gyp ERR! System Darwin 19.6.0
gyp ERR! command "/Users/sanak/.nvm/versions/node/v14.17.0/bin/node" "/Users/sanak/.nvm/versions/node/v14.17.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "build" "--fallback-to-build" "--module=/Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents/lib/binding/Release/node-v83-darwin-x64/fse.node" "--module_name=fse" "--module_path=/Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents/lib/binding/Release/node-v83-darwin-x64" "--napi_version=8" "--node_abi_napi=napi"
gyp ERR! cwd /Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents
gyp ERR! node -v v14.17.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/Users/sanak/.nvm/versions/node/v14.17.0/bin/node /Users/sanak/.nvm/versions/node/v14.17.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents/lib/binding/Release/node-v83-darwin-x64/fse.node --module_name=fse --module_path=/Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents/lib/binding/Release/node-v83-darwin-x64 --napi_version=8 --node_abi_napi=napi' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:376:20)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1055:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
node-pre-gyp ERR! System Darwin 19.6.0
node-pre-gyp ERR! command "/Users/sanak/.nvm/versions/node/v14.17.0/bin/node" "/Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents/node_modules/node-pre-gyp/bin/node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd /Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents
node-pre-gyp ERR! node -v v14.17.0
node-pre-gyp ERR! node-pre-gyp -v v0.10.0
node-pre-gyp ERR! not ok 
Failed to execute '/Users/sanak/.nvm/versions/node/v14.17.0/bin/node /Users/sanak/.nvm/versions/node/v14.17.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents/lib/binding/Release/node-v83-darwin-x64/fse.node --module_name=fse --module_path=/Users/sanak/Projects/gtt-project/git/ol-ext/node_modules/fsevents/lib/binding/Release/node-v83-darwin-x64 --napi_version=8 --node_abi_napi=napi' (1)
npm WARN ol-ext@3.2.3 requires a peer of ol@>= 5.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 install: `node install`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1

added 152 packages from 69 contributors, removed 171 packages, updated 572 packages and audited 795 packages in 18.774s

3 packages are looking for funding
  run `npm fund` for details

found 632 vulnerabilities (458 low, 18 moderate, 155 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
```

**Comment by sanak** [comment]
Here is the latter results after above error.
(I used `npx` command to run gulp on https://github.com/Viglino/ol-ext/blob/master/DEVELOPING.md doc.)
```bash
$ npx gulp
[15:31:51] Using gulpfile ~/Projects/gtt-project/git/ol-ext/gulpfile.js
[15:31:51] Starting 'default'...
[15:31:51] Starting 'js'...
[15:31:51] Starting 'extrajs'...
[15:31:51] Starting 'css'...
[15:31:51] Starting 'css0'...
[15:31:51] Starting 'cssmin'...
[15:31:51] Finished 'extrajs' after 236 ms
[15:31:51] Finished 'css0' after 350 ms
[15:31:52] Finished 'cssmin' after 583 ms
[15:31:52] Finished 'css' after 604 ms
 
>>> Terminated... 
[15:31:55] Finished 'js' after 4.32 s
[15:31:55] Finished 'default' after 4.32 s

$ npx gulp lib
[15:32:02] Using gulpfile ~/Projects/gtt-project/git/ol-ext/gulpfile.js
[15:32:02] Starting 'lib'...
[15:32:02] The following tasks did not complete: lib
[15:32:02] Did you forget to signal async completion?

$ npm run lint

> ol-ext@3.2.3 lint /Users/sanak/Projects/gtt-project/git/ol-ext
> eslint --fix src/ || exit 0

$ npx gulp doc 
[15:32:41] Using gulpfile ~/Projects/gtt-project/git/ol-ext/gulpfile.js
[15:32:41] Starting 'doc'...
[15:32:41] Finished 'doc' after 53 ms
[15:32:43] ERROR: Unable to parse a tag's type expression for source file /Users/sanak/Projects/gtt-project/git/ol-ext/dist/ol-ext.js in line 31182 with tag title "param" and text "{ol.coordinate||Array<ol.coordinate>} coord coordinate or an array of coordinates": Invalid type expression "ol.coordinate||Array<ol.coordinate>": Expected "!", "$", "'", "*", ".", "...", "0", "?", "@", "Function", "\"", "\\", "_", "break", "case", "catch", "class", "const", "continue", "debugger", "default", "delete", "do", "else", "enum", "export", "extends", "false", "finally", "for", "function", "if", "implements", "import", "in", "instanceof", "interface", "let", "new", "null", "package", "private", "protected", "public", "return", "static", "super", "switch", "this", "throw", "true", "try", "typeof", "undefined", "var", "void", "while", "with", "yield", "{", Unicode letter number, Unicode lowercase letter, Unicode modifier letter, Unicode other letter, Unicode titlecase letter, Unicode uppercase letter, or [1-9] but "|" found.

[15:32:43] ERROR: Unable to parse a tag's type expression for source file /Users/sanak/Projects/gtt-project/git/ol-ext/dist/ol-ext.js in line 31190 with tag title "param" and text "{ol.coordinate||Array<ol.coordinate>} coord coordinate or an array of coordinates": Invalid type expression "ol.coordinate||Array<ol.coordinate>": Expected "!", "$", "'", "*", ".", "...", "0", "?", "@", "Function", "\"", "\\", "_", "break", "case", "catch", "class", "const", "continue", "debugger", "default", "delete", "do", "else", "enum", "export", "extends", "false", "finally", "for", "function", "if", "implements", "import", "in", "instanceof", "interface", "let", "new", "null", "package", "private", "protected", "public", "return", "static", "super", "switch", "this", "throw", "true", "try", "typeof", "undefined", "var", "void", "while", "with", "yield", "{", Unicode letter number, Unicode lowercase letter, Unicode modifier letter, Unicode other letter, Unicode titlecase letter, Unicode uppercase letter, or [1-9] but "|" found.

[15:32:46] JSDoc returned with error code: 1
```

**Comment by sanak** [comment]
Well, about 1st comment error, I fixed it in #618, but `npx gulp lib` and `npx gulp doc` still report above errors, so I will create another issues for those.

**Comment by Viglino** [comment]
You have to use npx gulp if your global version of gulp is different of the project version...
